### PR TITLE
Add camel-azure-cosmosdb wrapper

### PR DESCRIPTION
**Pull request by lopushen**

### CODE REVIEW

**Review by f2par0 — APPROVED** [review]
- no conflict :heavy_check_mark: 
- compilation works :heavy_check_mark: 
- no unnecessary usage of wrap protocol :heavy_check_mark: 
- no unnecessary dependencies :heavy_check_mark: 
- no hardcoded version if it's in the parent pom :heavy_check_mark: 
- feature is placed in alphabetical order in the file camel-features.xml  :heavy_check_mark: 
- usage of dependent features (netty, jackson, jakarta ...) :heavy_check_mark:

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -477,6 +477,17 @@
...
     </feature>
+    <feature name='camel-azure-cosmosdb' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='[2.16,2.17)'>jackson</feature>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-core/1.46.0</bundle>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-cosmos/4.56.0</bundle>
+        <bundle dependency='true'>wrap:mvn:com.azure/azure-identity/1.11.2</bundle>
+        <bundle dependency='true'>mvn:org.reactivestreams/reactive-streams/${reactive-streams-version}</bundle>
+        <bundle dependency='true'>mvn:io.projectreactor/reactor-core/3.4.34</bundle>
+        <bundle dependency='true'>mvn:org.apache.camel.karaf/camel-cloudevents/${project.version}</bundle>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]: This should be a feature, it makes me realize that we have the same mistake at several places, could you please fix it too

Comment (lopushen) [reply features/src/main/feature/camel-features.xml]: fixed, used `camel-cloudevents` instead of `camel-core`

**Review by essobedo — APPROVED** [review]
LGTM

**Pull request merged**